src/OrchardCore.Modules/OrchardCore.ContentFields/GraphQL/Fields/ContentFieldsProvider.cs

```diff
@@ -81,7 +82,8 @@ public class ContentFieldsProvider : IContentFieldProvider
                     Description = "Multi text field",
                     FieldType = typeof(ListGraphType<StringGraphType>),
                     UnderlyingType = typeof(MultiTextField),
-                    FieldAccessor = field => (string[]) field.Content.Values
+                    // (string[]) can not be converted directly from content.values
+                    FieldAccessor = field => ((JArray)field.Content.Values)?.ToObject<string[]>()
```

Comment:
ok you can remove the comment .. these newtsonsoft refs make me cry tho .. we gotta sortt it out been system.text.json compliant at some point :P (its fine cos there is other stuff like this already)

Reply:
I was about to say the same thing before seeing the comment. We shouldn't tie it to NewtonSoft. If the field is `MultiTextField` then cast to that, or map to it, and access its own properties.